xml/cha_introduction.xml

```diff
@@ -296,4 +296,52 @@
   </important>
  </sect1>
 
+ <sect1 xml:id="sec-intro-support">
+  <title>Support</title>
+  <para>
+   Regardless of the public cloud you use and the plan you choose, &suse; has
+   you covered with support. Details however differ:
+  </para>
+  <itemizedlist>
+   <listitem>
+    <para>
+     PAYG instances include support through the cloud provider which in turn is
+     supported by &suse;. The CSP provides 1st and 2nd level, &suse; will
+     provide level 3 if necessary.
```

Comment:
The support story gets way more complicated. Principally we could state that products resold by the provider get L1 & L2 support from the provider. But of course that leave the reader/customer hanging with the question what is an what is not resold by the CSP, for example in AWS there is not really a good way for the customer to figure that out for things found in the general catalog. Then again, if it's in the general catalog it's resold or it's BYOS, so that rule is reasonably easy to express. In any event this is going to be CSP specific and a general statement that is simple is not going to reflect the world. We can decide to try and describe the situation as it is, for Google it's simple, everything is L1 & L2 supported by them except for BYOS, or we can bit the bullet and list support by image category. For example SLES & SLES4SA PAYG in AWS are L1 & L2 supported by AWS, but SLE HPC (when we list it) will be L1, L2, & L3 supported by SUSE.

Reply:
Also part of the support story are those customers that use PAYG but also have a direct relationship with SUSE. They get L1 & L2 from us even if they run PAYG that is nominally supported by the CSP for L1 & L2